subtests/docker_cli/selinux_labels/selinux_labels.py

```diff
@@ -0,0 +1,46 @@
+r"""
+Summary
+---------
+
+Verify that docker, daemons, and containers all have expected
+selinux lables.
+
+Operational Summary
+----------------------
+
+#. Execute bundled test.sh script
+#. Fail subtest if script returns non-zero
+
+Prerequisites
+---------------
+
+Commands contained w/in test script are available/functional on system.
+"""
+
+from os.path import join
+from autotest.client.utils import run
+from dockertest import subtest
+from dockertest.images import DockerImages
+from dockertest.output.validate import mustpass
+
+
+class selinux_labels(subtest.Subtest):
+
+    def initialize(self):
```

Comment:
probably needs `DockerVersion.require_server("1.12")` (dockerd and containerd don't exist in 1.10)